### ISSUE TITLE
Android RNIapModule.getItems() method response 404 caused app crash

### ISSUE THREAD

**Issue by tehong** [dooboolab/react-native-iap#35]
```
java.lang.NullPointerException Attempt to invoke virtual method 'java.lang.String java.lang.Object.toString()' on a null object reference 
    RNIapModule.java:134 com.reactlibrary.RNIapModule$3.onSkuDetailsResponse
    BillingClientImpl.java:404 com.android.billingclient.api.BillingClientImpl.querySkuDetailsAsync
    RNIapModule.java:129 com.reactlibrary.RNIapModule.getItems
    Method.java:-2 java.lang.reflect.Method.invoke
    JavaMethodWrapper.java:374 com.facebook.react.bridge.JavaMethodWrapper.invoke
    JavaModuleWrapper.java:162 com.facebook.react.bridge.JavaModuleWrapper.invoke
    NativeRunnable.java:-2 com.facebook.react.bridge.queue.NativeRunnable.run
    Handler.java:751 android.os.Handler.handleCallback
    Handler.java:95 android.os.Handler.dispatchMessage
    MessageQueueThreadHandler.java:31 com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage
    Looper.java:154 android.os.Looper.loop
    MessageQueueThreadImpl.java:194 com.facebook.react.bridge.queue.MessageQueueThreadImpl$3.run
    Thread.java:762 java.lang.Thread.run
```

Looking at the code in RNIapModule.java:

```
@ReactMethod
  public void getItems(String items, final Callback cb) {
    if (mService == null) {
      cb.invoke("IAP not prepared. Check if google play service is available.", null);
      return;
    }

    try {
      JSONArray jsonArray = new JSONArray(items);
      ArrayList<String> skuList = new ArrayList<> ();

      for (int i = 0; i < jsonArray.length(); i++) {
        String str = jsonArray.get(i).toString();
        skuList.add(str);
      }

      SkuDetailsParams.Builder params = SkuDetailsParams.newBuilder();
      params.setSkusList(skuList).setType(BillingClient.SkuType.INAPP);
      mBillingClient.querySkuDetailsAsync(params.build(),
          new SkuDetailsResponseListener() {
            @Override
            public void onSkuDetailsResponse(int responseCode, List<SkuDetails> skuDetailsList) {
              Log.d(TAG, "responseCode: " + responseCode);
              Log.d(TAG, skuDetailsList.toString());     
```

It appears that toString() method on the last line caused the crash.  I believe the HTTP response was 404 Not Found based on the crash log.

**Comment by dooboolab** [comment]
Thank you for the notice. I will debug it soon. Could you tell me which version of react-native-iap you are using?

**Comment by dooboolab** [comment]
@tehong I've just published the module to npm `0.2.13`. Please review `breaking changes` section. Also I applied changes for your crash log. Thank you.

**Comment by tehong** [comment]
@dooboolab My version is ```0.2.9```.  I see that you've made some changes to the latest ```0.2.13``` that fixes this problem.  I will upgrade to that and thanks for the ```breaking changes``` reminder. Most of all,  thanks for the quick fix.
